app/containers/SignIn/SignIn.jsx

```diff
@@ -51,7 +50,7 @@ class SignIn extends React.Component {
 
         case 501:
           message =
-            'The API is running an earlier version than that required by this version of Publish'
+            'The API is running an earlier version than that required by this version of Edit'
```

Comment:
"…version of Edit **Interface**", I'd say, because the API is also Edit. Everything is Edit now :stuck_out_tongue:

Reply:
True. Although we need a global review of text and terminology, so I won't worry too much about it for now.